### ISSUE TITLE
Calculating corrected intersection losses at inference

### ISSUE THREAD

**Issue by J-Salo** [HannesStark/EquiBind#27]
Hi,

When performing inference (from files), the untuned losses are calculated and reported in `all_intersection_losses_untuned`. However, the optimized losses never seem to be calculated and reported.

In `inference.py` can we use something a long the lines of:
```
all_intersection_losses_untuned.append(
                    compute_revised_intersection_loss(lig_coords_pred_untuned.detach().cpu(), rec_graph.ndata['x'],
                                                        alpha=0.2, beta=8, aggression=0))
# After calculating coords_pred_optimized:
all_intersection_losses.append(compute_revised_intersection_loss(coords_pred_optimized, rec_graph.ndata['x'], alpha=0.2, beta=8, aggression=0))
```

Many thanks!

**Comment by HannesStark** [comment]
Feel free to make these changes in your repo. I do not think this is important. Let me know if you disagree!

**Comment by hmms117** [comment]
Thanks Hannes - will do :) 
I had hoped the loss of the optimized ligand would be a good rough proxy for fit (as an initial criteria). I guess the tuned loss would be good enough as well.
